src/main/java/Deadline.java

```diff
@@ -0,0 +1,12 @@
+public class Deadline extends Task{
+    private String by;
```

Comment:
perhaps a different name for this variable? 
by doesn't really mean anything by itself